scripts/CEE/hs-must-gather/script.sh

```diff
@@ -4,46 +4,30 @@ set -e
 set -o nounset
 set -o pipefail
 
-#define vars 
-HS_HCP_MG_DIR="$DUMP_DIR/managment-cluster"
-HS_DP_MG_DIR="$DUMP_DIR/hs-cluster"
-HS_BINARY_PATH=" /usr/local/bin/hypershift"
-
- # check hypershift binary
-function hasHipershiftBinary {
-    echo "Checking hypershift binary available in $HS_BINARY_PATH" 
-    if test -f "$HS_BINARY_PATH";
-     then
-        return 1
-     else
-        return 0
-    fi
-   
-}
-
-# Init must gather for Hosted controll plane
-function initMustGatherforHCP {
-    echo "Init must gather for management cluster namespace $HS_NAMESPACE, and data will saved to $HS_HCP_MG_DIR" 
-
-    return 1
-}
-
-# Init must gather for hosted cluster
-function initMustGatherforHC {
-    echo "Init must gather for hosted cluster $CLUSTER_NAME, and data will saved to $HS_DP_MG_DIR" 
-
-    return 1
-}
-
-# Get 
-if hasHipershiftBinary "$1" ; then
-    if initMustGatherforMC "$1" ; then
-        initMustGatherforHC
-    fi
-
-    
-else
-    echo "hypershift binary not available in $HS_BINARY_PATH path" 
+# Define expected values 
+HS_BINARY_PATH="/usr/local/bin/hypershift"
+
+# Check if hypershift binary is in place 
+if [ ! -f "$HS_BINARY_PATH" ]; then
+  echo "HyperShift binary not found in $HS_BINARY_PATH path" 
+  exit 1
+fi
+
+# Sanity check for multiple namespaces matching label
+HCP_NS_LIST_LENGTH=$(oc get namespace -l api.openshift.com/id="$CLUSTER_ID" -o json | jq -r '.items | length')
+if [ "$HCP_NS_LIST_LENGTH" -ne 1 ]; then
```

Comment:
I am not sure if this check is needed, since the cluster ID should be unique, but as we have two namespaces for each cluster, will the list with label `api.openshift.com/id="$CLUSTER_ID"` return both of them?

Reply:
Had a check, it will return the namespace without cluster name only.
So I do not think it is necessary to check namespace count with the label?

Reply:
+1 this check isn't needed anymore

Reply:
yeah, If it includes the condition where the cluster not found, that makes sense to me. And then we should update the comments.